misk/src/test/kotlin/misk/web/interceptors/MetricsInterceptorTest.kt

```diff
@@ -54,18 +63,28 @@ class MetricsInterceptorTest {
   }
 
   fun invoke(desiredStatusCode: Int): Response<String> {
+    val request = Request(
+        HttpUrl.parse("http://foo.bar/")!!,
```

Comment:
Off topic: pretty much everyone in the world hates me because parse() can return null

Reply:
Not really that big a deal in Kotlin, although if you don't know the language I guess you could think I REALLY want it parsed.